modules/textures/src/compressed-texture-loader.ts

```diff
@@ -37,7 +40,13 @@ export const CompressedTextureWorkerLoader = {
  */
 export const CompressedTextureLoader = {
   ...CompressedTextureWorkerLoader,
-  parse: async (arrayBuffer) => parseCompressedTexture(arrayBuffer)
+  parse: async (arrayBuffer, options) => {
+    if (options['compressed-texture'].useBasis) {
+      options.basis = {...options.basis, decoderFormat: 'KTX2', module: 'encoder'};
```

Comment:
Can we use a more standard way of applying default options, see other loaders?

Reply:
There could be values from user with custom `format`